pkg/antctl/raw/packetcapture/command.go

```diff
@@ -0,0 +1,327 @@
+  Save the packets file to a specified directory
+  $ antctl packetcapture -S 192.168.123.123 -D pod2 -f tcp,tcp_dst=80 -o /tmp
+`)
+
+func init() {
+	Command = &cobra.Command{
+		Use:     "packetcapture",
+		Short:   "Start capture packets",
+		Long:    "Start capturing packets on the target flow",
+		Aliases: []string{"pc", "packetcaptures"},
+		Example: packetCaptureExample,
+		RunE:    packetCaptureRunE,
+	}
+
+	Command.Flags().StringVarP(&option.source, "source", "S", "", "source of the the PacketCapture: Namespace/Pod, Pod, or IP")
+	Command.Flags().StringVarP(&option.dest, "destination", "D", "", "destination of the PacketCapture: Namespace/Pod, Pod, or IP")
+	Command.Flags().Int32VarP(&option.number, "number", "n", 0, "target number of packets to capture, the capture will stop when it is reached")
+	Command.Flags().StringVarP(&option.flow, "flow", "f", "", "specify the flow (packet headers) of the PacketCapture, including tcp_src, tcp_dst, udp_src, udp_dst")
+	Command.Flags().BoolVarP(&option.nowait, "nowait", "", false, "if set, command returns without retrieving results")
+	Command.Flags().StringVarP(&option.outputDir, "output-dir", "o", ".", "save the packets file to the target directory")
+}
+
+var protocols = map[string]int32{
+	"icmp": 1,
+	"tcp":  6,
+	"udp":  17,
+}
+
+func getFlowFields(flow string) (map[string]int, error) {
+	fields := map[string]int{}
+	for _, v := range strings.Split(flow, ",") {
+		kv := strings.Split(v, "=")
+		if len(kv) == 2 && len(kv[0]) != 0 && len(kv[1]) != 0 {
+			r, err := strconv.Atoi(kv[1])
+			if err != nil {
+				return nil, err
+			}
+			fields[kv[0]] = r
+		} else if len(kv) == 1 {
+			if len(kv[0]) != 0 {
+				fields[v] = 0
+			}
+		} else {
+			return nil, fmt.Errorf("%s is not valid in flow", v)
+		}
+	}
+	return fields, nil
+}
+
+func getPodFile(cmd *cobra.Command) (PodFileCopy, error) {
+	config, client, _, err := getClients(cmd)
+	if err != nil {
+		return nil, err
+	}
+	return &podFile{
+		restConfig: config,
+		client:     client,
+	}, nil
+}
+
+func getConfigAndClients(cmd *cobra.Command) (*rest.Config, kubernetes.Interface, antrea.Interface, error) {
+	kubeConfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+	k8sClientset, client, err := raw.SetupClients(kubeConfig)
+	if err != nil {
+		return nil, nil, nil, fmt.Errorf("failed to create clientset: %w", err)
+	}
+	return kubeConfig, k8sClientset, client, nil
+}
+
+func getPCName(src, dest string) string {
+	replace := func(s string) string {
+		return strings.ReplaceAll(s, "/", "-")
+	}
+	prefix := fmt.Sprintf("%s-%s", replace(src), replace(dest))
+	if option.nowait {
+		return prefix
+	}
+	return fmt.Sprintf("%s-%s", prefix, rand.String(8))
+}
+
+func packetCaptureRunE(cmd *cobra.Command, args []string) error {
+	option.timeout, _ = cmd.Flags().GetDuration("timeout")
+	if option.timeout > 300*time.Second {
+		return errors.New("timeout cannot be longer than 1 hour")
```

Comment:
it's 5 minutes, not 1 hour

maybe define a constant `maxPacketCaptureTimeout` and use it in both places (the check and the error message)